README.md

```diff
@@ -20,7 +20,7 @@ To do this, add the dependency to your pom.xml:
     <dependency>
         <groupId>com.maxmind.geoip2</groupId>
         <artifactId>geoip2</artifactId>
-        <version>2.2.0</version>
+        <version>2.3.0</version>
```

Comment:
We should modify the release script to do this automatically if it doesn't already. If we do it before the release, people will follow the directions on GitHub and then email support about it not working.